backend/protocol_rpc/endpoint_generator.py

```
@@ -79,6 +79,70 @@ def handle_eth_methods():
+                            with requests.Session() as http:
+                                result = http.post(
+                                    HARDHAT_URL,
+                                    json=request_json,
+                                    headers={"Content-Type": "application/json"},
+                                )
+                                return flask.Response(
+                                    result.content,
+                                    status=result.status_code,
+                                    headers=dict(result.headers),
+                                )
+                        except requests.RequestException as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)